src/setup.jl

```diff
@@ -35,6 +37,12 @@ function GenerateGrid(
 
 end
 
+function calcF!(Fch, sol, t, clock, vars, params, grid)
+    Fch .= 0
+
+    return nothing
+end
```

Comment:
at the moment `Fch` is zeros.... you should modify this so that it computes `Fch` given `sol`, `vars`, `param`, and the `grid`.